src/main/java/frc/robot/subsystems/Revolver.java

```diff
@@ -105,7 +116,7 @@ public void stop() {
 
   @Override
   public void periodic() {
-    if (m_repsAboveFeed > Constants.REVOLVER_REPS_THRESHOLD || m_repsAboveDrum > Constants.REVOLVER_REPS_THRESHOLD) {
+    if (m_drumJammed || m_feedJammed) {
       new UnjamRevolver(this).schedule(false);
     }
```

Comment:
Could be implemented in the set method instead of in periodic

Reply:
Personally I think it makes more sense in `periodic` since it has more to do with constantly monitoring the state of the revolver than with setting the revolver speed.

Reply:
that makes sense, alright